tests/monitoring_test.go

```diff
@@ -92,40 +92,29 @@ var _ = Describe("Exporters", func() {
 	})
 
 	DescribeTable("Exported timeseries", func(selector string, match types.GomegaMatcher) {
+		var series []map[string]string
 		params := map[string]string{
 			"match[]": selector,
 			"start":   fmt.Sprintf("%d", time.Now().Add(-20*time.Minute).Unix()),
 		}
-		resultRaw, err := c.CoreV1().
-			Services("kubeprod").
-			ProxyGet("http", "prometheus", "9090", "api/v1/series", params).
-			DoRaw()
-		Expect(err).NotTo(HaveOccurred())
 
-		resp := promResponse{}
-		json.Unmarshal(resultRaw, &resp)
-		var series []map[string]string
-		json.Unmarshal(resp.Data, &series)
-
-		fmt.Fprintf(GinkgoWriter, "%s found %d timeseries:\n", selector, len(series))
-		for i, s := range series {
-			if i >= 10 {
-				fmt.Fprintf(GinkgoWriter, "(truncated ...)\n")
-				break
-			}
-			fmt.Fprintf(GinkgoWriter, "%s{", s["__name__"])
-			for k, v := range s {
-				if k != "__name__" {
-					fmt.Fprintf(GinkgoWriter, "%s=%q,", k, v)
-				}
-			}
-			fmt.Fprintf(GinkgoWriter, "}\n")
-		}
-		Eventually(func() bool {
-			return Expect(series).To(match)
-		}, "2m", "5s").Should(BeTrue())
+		Eventually(func() ([]map[string]string, error) {
+			resultRaw, err := c.CoreV1().
+				Services("kubeprod").
+				ProxyGet("http", "prometheus", "9090", "api/v1/series", params).
+				DoRaw()
+			Expect(err).NotTo(HaveOccurred())
+
+			resp := promResponse{}
+			json.Unmarshal(resultRaw, &resp)
+			json.Unmarshal(resp.Data, &series)
+
+			fmt.Printf("%v\n", series)
```

Comment:
Don't write to stdout, write to `GinkgoWriter`.  That way the logging output is correctly associated with the test, and appears only if `ginkgo -v` is used.

Reply:
oops! this was not supposed to be committed